File: app/src/main/java/com/app/rupiksha/models/CMSReportModel.java
package com.app.rupiksha.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CMSReportModel
{
    @SerializedName("id")
    @Expose
    private int id;
    @SerializedName("uid")
    @Expose
    private String uid;
    @SerializedName("txnid")
    @Expose
    private String txnid;
    @SerializedName("amount")
    @Expose
    private String amount;
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("operator_name")
    @Expose
    private String operatorName;
    @SerializedName("req")
    @Expose
    private String req;
    @SerializedName("res")
    @Expose
    private String res;
    @SerializedName("debitreq")
    @Expose
    private Object debitreq;
    @SerializedName("callback")
    @Expose
    private Object callback;
    @SerializedName("date")
    @Expose
    private String date;

    private boolean progressDelay=false;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getTxnid() {
        return txnid;
    }

    public void setTxnid(String txnid) {
        this.txnid = txnid;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getOperatorName() {
        return operatorName;
    }

    public void setOperatorName(String operatorName) {
        this.operatorName = operatorName;
    }

    public String getReq() {
        return req;
    }

    public void setReq(String req) {
        this.req = req;
    }

    public String getRes() {
        return res;
    }

    public void setRes(String res) {
        this.res = res;
    }

    public Object getDebitreq() {
        return debitreq;
    }

    public void setDebitreq(Object debitreq) {
        this.debitreq = debitreq;
    }

    public Object getCallback() {
        return callback;
    }

    public void setCallback(Object callback) {
        this.callback = callback;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public boolean isProgressDelay() {
        return progressDelay;
    }

    public void setProgressDelay(boolean progressDelay) {
        this.progressDelay = progressDelay;
    }
}

File: app/src/main/java/com/app/rupiksha/models/CMSReportModel.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class CMSReportModel(
    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("uid")
    @Expose
    val uid: String? = null,

    @SerializedName("txnid")
    @Expose
    val txnid: String? = null,

    @SerializedName("amount")
    @Expose
    val amount: String? = null,

    @SerializedName("status")
    @Expose
    val status: String? = null,

    @SerializedName("operator_name")
    @Expose
    val operatorName: String? = null,

    @SerializedName("req")
    @Expose
    val req: String? = null,

    @SerializedName("res")
    @Expose
    val res: String? = null,

    @SerializedName("debitreq")
    @Expose
    val debitreq: Any? = null,

    @SerializedName("callback")
    @Expose
    val callback: Any? = null,

    @SerializedName("date")
    @Expose
    val date: String? = null,

    var progressDelay: Boolean = false
)
